test_elasticsearch/README.rst

```diff
@@ -39,6 +39,9 @@ Alternatively, if you wish to control what you are doing you have several additi
  * you can just run your favorite runner in the ``test_elasticsearch`` directory
    (verified to work with nose and py.test) and bypass the fetch logic entirely.
 
+ * to run a specific test, you can use ``python3 setup.py test -s <test_name>``, for example
+   ``python3 setup.py test -s test_elasticsearch.test_helpers.TestParallelBulk.test_all_chunks_sent``
```

Comment:
it might be better to specify how to work with the `run_tests.py` which allows more option and is faster as it skips the initialization of the project which is only needed once (and not needed if you do `pip install -e '.[develop]'`). You can run individual files and/or tests, only previously failed tests etc (see https://nose.readthedocs.io/en/latest/man.html for details)

Reply:
i haven't been able to run run_tests.py on its own, when I try i get 
`Traceback (most recent call last):
  File "test_elasticsearch/run_tests.py", line 85, in <module>
    run_all(sys.argv)
  File "test_elasticsearch/run_tests.py", line 65, in run_all
    fetch_es_repo()
  File "test_elasticsearch/run_tests.py", line 38, in fetch_es_repo
    from test_elasticsearch.test_server import get_client
ImportError: No module named test_elasticsearch.test_server`
However, it works for me when called through setup.py.  Do you know why this might happen?

Reply:
How have you installed the library, did you run the `pip` command?

Reply:
ah running `pip install -e .[develop]` fixed it, I had switched dev machines and forgot to re-run it.